content/curating/guidelines/kata.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar: curating
+prev: /curating/guidelines/
+next: /curating/guidelines/translation/
+---
+
+
+# Kata Curating Guidelines
+
+This article contains a set of guidelines, which can be used by reviewers to determine whether a kata meets quality requirements to be approved and become available for users to train on. They were collected to help ensure that kata are of sufficient quality and users' experience will be as good as possible.
+
+## Review
+
+- **The kata must be reviewed before it's approved.** See [Curating a Kata HOWTO][howto-review-kata] for more details on how to review a kata.
+- **The review should be as complete as possible,** including all applicable steps mentioned in [Curating a Kata HOWTO][howto-review-kata]. If the reviewer is not able to perform a complete review, they should ask other users for help.
+- **Conformity to [kata authoring guidelines][guidelines-authoring-kata] should be verified.** All applicable guideliness should be followed by the author.
+
+
+## Feedback
+
+- **The satisfaction rating should indicate quality of kata as a concept, not its current state of condition in terms of open issues which can be fixed.** If you think the kata itself is valuable but you simply have some issues with it, then you should select "Somewhat" and create an issue instead. If you think the kata is really great, other than the issues, then you can choose "Very". Do not choose "None" simply because the kata has an issue but it is otherwise a good kata.
+- **When casting a vote for difficulty rating, reviewers should consider broader picture than just their own skills.** It's true that the way how difficulty rankings are handled by Codewars is far from perfect, and not balanced well. It's also true that difficulty level is not easy to estimate, because what is easy for one, can be difficult for another, and vice versa. The way how difficulty ranking works will be hopefully improved in the future, but until then, your votes should not throw the system deliberately off balance.
```

Comment:
```suggestion
- **When casting a vote for difficulty rating, reviewers should consider a broader picture than just their own skills.** It's true that the way how difficulty rankings are handled by Codewars is far from perfect, and not balanced well. It's also true that difficulty level is not easy to estimate, because what is easy for one, can be difficult for another, and vice versa. The way how difficulty ranking works will be hopefully improved in the future, but until then, your votes should not throw the system deliberately off-balance.
```